src/ui_basic/dropdown.h

```diff
@@ -208,6 +214,12 @@ class BaseDropdown : public NamedPanel {
 	/// Returns true if the mouse pointer left the vicinity of the dropdown.
 	bool is_mouse_away() const;
 
+	virtual void apply_filter() = 0;
+
+	virtual void save_selected_entry(uint32_t){
+	   // cannot pure because it is called from constructor
+	};
```

Comment:
Clang-tidy and our self-written codecheck rules don't always like each other…

Usually a comment does the trick:
```suggestion
	virtual void save_selected_entry(uint32_t)  // cannot be pure virtual because it is called from constructor
	{
	};
```